app/source/components/stats-table/stats-table.scss

```diff
@@ -0,0 +1,119 @@
+//table color scale based on percentage of max value per row (category).
+$base-color: #268ebe;
```

Comment:
Bør bruke samme farger som resten av siten. Det er spesifisert fargekoder i `vars.scss`. Hvis du trenger en mellomting mellom de to blåfargene kan du bruke `mix($blue, $bright-blue)`